tests/query_indexes.rs

```diff
@@ -175,28 +98,28 @@ pub async fn test_should_get_indexes_with_context(cluster: Arc<ClusterUnderTest>
             );
         },
     );
-
-    false
 }
 
-pub async fn test_should_get_indexes(cluster: Arc<ClusterUnderTest>) -> bool {
-    let config = cluster.config();
-    if !config.supports_feature(TestFeature::QueryIndex) {
-        return true;
-    }
+#[test]
+#[cfg_attr(not(feature = "query_index"), ignore)]
+fn list_indexes() {
+    let config = utils::test_cluster().config();
 
-    CBPlayground::setup(
-        "test_should_get_indexes",
-        cluster.config(),
+    playground::CBPlayground::setup(
+        "list_indexes",
+        None,
         PerTestOptions::default().set_no_default_collection(true),
         |dirs, sandbox| {
             let keyspace = config.bucket();
             let cmd = "".to_string();
             let fields = "`field1`,`field2`";
             let index_name1 =
-                create_index(cmd.clone(), fields, keyspace.clone(), dirs.test(), sandbox);
+                utils::create_index(cmd.clone(), fields, keyspace.clone(), dirs.test(), sandbox);
             let index_name2 =
-                create_index(cmd.clone(), fields, keyspace.clone(), dirs.test(), sandbox);
+                utils::create_index(cmd.clone(), fields, keyspace.clone(), dirs.test(), sandbox);
+
+            // Let the indexes be created
+            sleep(time::Duration::from_secs(20));
```

Comment:
We should use a consistency util rather than a sleep.

Reply:
Done!